src/components/widgets/sales-opportunities-funnel/sales-opportunities-funnel.directive.coffee

```diff
@@ -137,6 +137,7 @@ module.controller('WidgetSalesOpportunitiesFunnelCtrl', ($scope, $q, ChartFormat
       sortedOppGroups = []
       angular.forEach $scope.assigneesOptions, (assigneeOption) ->
         assigneeId = assigneeOption.value
+        assigneeOption.label = "Not assigned" if assigneeOption.label.toLowerCase() == "unknown"
```

Comment:
wouldn't it be simpler to return it from the api?

Reply:
@cesar Very easy to make the change in Impac!